ios/Classes/components/card/session/CardSessionComponent.swift

```diff
@@ -42,31 +42,18 @@ class CardSessionComponent: BaseCardComponent {
         guard let cardComponentConfiguration else { throw PlatformError(errorDescription: "Card configuration not found") }
```

Comment:
I see most parts of this function are repeated for advanced flow. Could be extracted into a single function except the session lines.